README.md

```diff
@@ -59,3 +59,44 @@ The project currently has a `GET hello-world` Api Gateway<>Lambda integration se
    ```
    yarn test:integ
    ```
+
+## Webhook Quoting Schema
+
+Quoters will need to abide by the following schemas in order to successfully quote UniswapX orders.
+
+### Request
+
+This data will be included in the body of the request and will be sent to the given quote endpoint.
+
+```
+{
+   tokenInChainId: number,
+   tokenOutChainId: number,
+   requestId: string,
+   tokenIn: string,
+   tokenOut: string,
+   amount: string,
+   swapper: string,
+   type: string (ex. EXACT_INPUT or EXACT_OUTPUT),
+}
+```
+
+### Response
+
+This data will be expected in the body of the quote response.
+
+_Note: if a quoter elects to not quote a swap they should still send back a response but with a zero value in the `amountIn`/`amountOut` field, depending on the trade type._
```

Comment:
did we align on this vs. sending back status 204?

Reply:
Do we want to go with one or the other? I don't have a strong opinion here.

Reply:
Going to update this to say we accept either 404 or zero value but if we want to change that totally good with me. Will fix in future PR. Just want to get this out for quoters.